cmd/atlascmd/schema.go

```diff
@@ -88,7 +95,7 @@ databases), omit the relevant part from the url, e.g. "mysql://user:pass@localho
 To select specific schemas from the databases, users may use the "--schema" (or "-s" shorthand)
 flag.
 	`,
-		Run: CmdInspectRun,
+		RunE: CmdInspectRun,
```

Comment:
Why did you replace `Run` with `RunE`?

Reply:
Honestly just dislike cobra.CheckErr everywhere. Returning an error makes the code seem nicer to me.
Not strongly attached to this

Reply:
I don't mind, I wondered why it was needed in this PR. Let me know when it's ready and I'll review this